packages/desktop-gui/cypress/integration/project_nav_spec.coffee

```diff
@@ -232,6 +232,32 @@ describe "Project Nav", ->
         cy.get(".browsers-list")
           .find(".dropdown-toggle").should("not.be.visible")
 
+    describe "browser has a warning attached", ->
+      beforeEach ->
+        @browsers = [{
+          "name": "chromium",
+          "displayName": "Chromium",
+          "family": "chrome",
+          "version": "49.0.2609.0",
+          "path": "/Users/bmann/Downloads/chrome-mac/Chromium.app/Contents/MacOS/Chromium",
+          "majorVersion": "49",
+          "warning": "Cypress detected policy settings on your computer that may cause issues with using this browser. For more information, see https://on.cypress.io/bad-browser-policy"
+        }]
+
+        @config.browsers = @browsers
+        @openProject.resolve(@config)
+
+      it "shows warning icon with linkified tooltip", ->
+        cy.get(".browsers .fa-exclamation-triangle")
+          .then ($el) ->
+            $el[0].dispatchEvent(new Event("mouseover", {bubbles: true}))
```

Comment:
should just be using `cy.trigger()` here...

`cy.get("...").trigger("mouseover")`